addons/maaacks_game_template/base/scenes/Menus/MainMenu/MainMenu.gd

```diff
@@ -1,11 +1,17 @@
 class_name MainMenu
 extends Control
 
+const NO_VERSION_NAME : String = "0.0.0"
 
+## Defines the path to the game scene. Hides the play button if empty.
 @export_file("*.tscn") var game_scene_path : String
 @export var options_packed_scene : PackedScene
 @export var credits_packed_scene : PackedScene
-@export var version_name : String = '0.0.0'
+@export_group("Version")
+## Displays the value of `application/config/version`, set in project settings.
+@export var show_version : bool = true
```

Comment:
It's not a very useful flag (can just hide it in scene tree), but it does draw people's attention to the project setting.